dumper/dialer/ssh_test.go

```
@@ -34,3 +36,27 @@ func TestCreateSshClient(t *testing.T) {
+		}
+
+		if err := os.Remove(file.Name()); err != nil {
+			t.Error(err)
+		}
+	}()
+
+	_, err = parseSSHKey(file.Name())
+	assert.Nil(err)
+
+	base64Encoded := base64.StdEncoding.EncodeToString([]byte{'f'})
+	key, err := parseSSHKey(base64Encoded)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)